tests/integration/app_mock.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
So... umm, thinking about this even more. you can definitely use the existing app mock instead of adding an additional one with python:

Volume Mount the classpath and JAVA_HOME into an Alpine linux image? I think we already do something quite similar.